### PR TITLE
EZP-28266: Theme content edit and create in Admin UI according to guidelines

### PR DESCRIPTION
**Pull request by webhdx**
> JIRA: https://jira.ez.no/browse/EZP-28266

# Requires: 
* https://github.com/ezsystems/repository-forms/pull/175 
* https://github.com/ezsystems/repository-forms/pull/174 
* https://github.com/ezsystems/repository-forms/pull/173

# Description:
This PR introduces better design to:
* content edit
* content create
* draft create

### CODE REVIEW

File: src/bundle/Resources/config/services/menu.yml [hunk 1]
```diff
@@ -27,3 +27,14 @@ services:
         public: true
         tags:
             - { name: knp_menu.menu_builder, method: build, alias: ezplatform_admin_ui.menu.trash.sidebar_right }
+
+    EzSystems\EzPlatformAdminUi\Menu\ContentEditRightSidebarBuilder:
+        public: true
+        tags:
+            - { name: knp_menu.menu_builder, method: build, alias: ezplatform_admin_ui.menu.content_edit.sidebar_right }
+
+
```
Comment (sunpietro) [inline src/bundle/Resources/config/services/menu.yml]: extra line?

File: src/bundle/Resources/public/scss/_general.scss [hunk 2]
```diff
@@ -13,6 +13,12 @@ body {
     min-height: calc(100vh - 60px);
 }
 
+.ez-standalone-page {
+    .ez-main-row {
+        min-height: 100vh;
```
Comment (sunpietro) [inline src/bundle/Resources/public/scss/_general.scss]: Is it needed?

File: src/bundle/Resources/views/content/content_edit/content_create.html.twig [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+{% block details %}
+    <h4 class="text-muted">{{ 'creating'|trans({'%contentType%': contentType.name})|desc('Creating - %contentType%') }}</h4>
+    <h1>
+        <svg class="ez-icon ez-icon-{{ contentType.identifier }}">
+            <use xlink:href="{{ asset('bundles/ezplatformadminui/img/ez-icons.svg') }}#{{ contentType.identifier }}"></use>
+        </svg>
+        {{ 'new_content_item'|trans({'%contentType%': contentType.name})|desc('New %contentType%') }}
+    </h1>
+
+    <div class="small">
```
Comment (sunpietro) [inline src/bundle/Resources/views/content/content_edit/content_create.html.twig]: Maybe just `<small></small>` instead?

Comment (webhdx) [reply src/bundle/Resources/views/content/content_edit/content_create.html.twig]: Small is inline we need a block element here

File: src/bundle/Resources/public/scss/_general.scss (hunk 2 above)
Comment (webhdx) [reply src/bundle/Resources/public/scss/_general.scss]: Yes it is since we are hiding menu bar

**Pull request merged**